Source/UI/Coordinators/PasswordCoordinator.swift

```diff
@@ -194,4 +239,66 @@ extension PasswordCoordinator {
         }
         self.navigationController.pushViewController(viewController, animated: true)
     }
+    private func getPasswordFromKeychain(for identifier: Identifier) -> String? {
+        guard #available(iOS 11.3, *) else {
+            // Fallback to passsword login
+            return nil
+        }
+        var query = [String: Any]()
+        query[kSecClass as String] = kSecClassGenericPassword
+        query[kSecReturnData as String] = kCFBooleanTrue
+        query[kSecAttrAccount as String] = identifier.normalizedString as CFString
+        query[kSecAttrLabel as String] = "com.schibsted.account.biometrics.secrets" as CFString
+        query[kSecUseOperationPrompt as String] = "Please put your fingers on that button" as CFString
+
+        var queryResult: AnyObject?
+        let status = withUnsafeMutablePointer(to: &queryResult) {
+            SecItemCopyMatching(query as CFDictionary, UnsafeMutablePointer($0))
+        }
+
+        if status == noErr, let qresult =  queryResult as? Data {
+            let password = String(data: qresult as Data, encoding: .utf8)!
+            return password
+        } else {
+            return nil
+        }
+    }
+    private func updatekeyChain(for identifier: Identifier, loginFlowVariant: LoginMethod.FlowVariant, password: String, completion: @escaping () -> Void ) {
+        if #available(iOS 11.3, *) {
+            let accessControl = SecAccessControlCreateWithFlags(kCFAllocatorDefault, kSecAttrAccessibleWhenPasscodeSetThisDeviceOnly, .userPresence, nil)
+            var dictionary = [String: Any]()
+            dictionary[kSecClass as String] = kSecClassGenericPassword
+            dictionary[kSecAttrLabel as String] = "com.schibsted.account.biometrics.secrets" as CFString
```

Comment:
I see this string is used in multiple places. So maybe make this a constant up in the file somewhere? I guess the convention is either:
```
private let kSecAttrLabelValue = "blah"
```
or
```
fileprivate struct Constants {
  static let kSecAttrLabelValue = "blah"
}
```